ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
+import os
+import sys
+
+urls = urls_testing.urls
+number_of_processes = len(urls)
+timeout = 1
+
+
+def millis():
+    return int(round(time.time() * 1000))
+
+
+def get_uri_responses(urls):
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'
```

Comment:
Is it necessary to provide a user agent? As in: did you solve an actual problem by adding that?
If not, I'd probably not use it. 
Like this we tell the browser that we are an actual browser. But that is not the case, we are a script.
Nicer would be to have our own agent: https://de.wikipedia.org/wiki/User_Agent

Reply:
Yes, this was indeed needed for some pages. They will respond differently if a user agent is not provided (probably to prevent exactly what we are trying to do here).
Wouldn't it be reasonable if we'd use a very often used user agent, so it would be very stupid if a page blocked it?